contracts/andromeda_crowdfund/src/contract.rs

```diff
@@ -109,23 +109,48 @@ fn execute_mint(
         config.can_mint_after_sale || !sale_conducted,
         ContractError::CannotMintAfterSaleConducted {},
     )?;
+
+    let mission_contract = contract.get_mission_contract(deps.storage)?;
+    let token_contract =
+        config
+            .token_address
+            .get_address(deps.api, &deps.querier, mission_contract)?;
+    let crowdfund_contract = env.contract.address.to_string();
+
+    let mut resp = Response::new();
+    for mint_msg in mint_msgs {
```

Comment:
Do we need to consider a length cap on the amount of mint messages?

Reply:
Sure, I added a limit of 100.